--- origsrc/libssh2-1.4.3/configure.ac	2014-11-09 14:58:11.969241300 -0600
+++ src/libssh2-1.4.3/configure.ac	2014-11-09 15:19:02.011739600 -0600
@@ -116,10 +116,10 @@ AM_CONDITIONAL(LIBGCRYPT, test "$ac_cv_l
 
 # Not all OpenSSL have AES-CTR functions.
 if test "$ac_cv_libssl" = "yes"; then
-  save_LDFLAGS="$LDFLAGS"
-  LDFLAGS="$LDFLAGS $LIBSSL"
+  save_LIBS="$LIBS"
+  LIBS="$LIBS $LIBSSL"
   AC_CHECK_FUNCS(EVP_aes_128_ctr)
-  LDFLAGS="$save_LDFLAGS"
+  LIBS="$save_LIBS"
 fi
 
 # Look for Libz
